en/orm/database-basics.rst

```diff
@@ -894,7 +894,7 @@ If you are using a legacy schema that requires identifier quoting you can enable
 it using the ``quoteIdentifiers`` setting in your
 :ref:`database-configuration`. You can also enable this feature at runtime::
 
-    $conn->driver()->autoQuoting(true);
+    $conn->getDriver()->enableAutoQuoting(true);
```

Comment:
you can use  `enableAutoQuoting()` , no need to `true` , it is default value for it